src/ExtensionAttribute.cs

```diff
@@ -2,8 +2,10 @@
 
 #pragma warning disable 1591	// XML doc warning
 
+#if !NETCOREAPP2_0
```

Comment:
Why isn't this `NETSTANDARD2_0`?

Reply:
This is just the name of the compile flag inside the csproj file, automatically given by Rider while creating an empty project file. The name corresponds to the referenced platform, which is basically *Microsoft.NETCore.App/2.0.0*.